Gordon360/Models/ViewModels/RecIM/SeriesPatchViewModel.cs

```diff
@@ -7,8 +7,8 @@ namespace Gordon360.Models.ViewModels.RecIM
     public class SeriesPatchViewModel
     {
         public string? Name { get; set; }
-        public DateTime? StartDate { get; set; }
-        public DateTime? EndDate { get; set; }
+        public TimeOnly? StartDate { get; set; }
+        public TimeOnly? EndDate { get; set; }
```

Comment:
Okay so these are timeonly types now but their names are "*Date"? Can you rename the columns/properties pls?